plugin/trino-delta-lake/pom.xml

```diff
@@ -426,6 +431,15 @@
                             </ignoredResourcePatterns>
                         </configuration>
                     </plugin>
+
+                    <plugin>
```

Comment:
I need the `aws-java-sdk-core` dependency within the Glue tests. However I couldn't find a way to make `maven-dependency-plugin` accept this new dependency. Please advise whether is a better way to include it in the pom.xml file.

Reply:
what's the actual problem?